### ISSUE TITLE
Failed to  connect to any SSH Host, alerting "Running the contributed command: 'opensshremotes.openEmptyWindow' failed“.

### ISSUE THREAD

**Issue by youralmight** [microsoft/vscode-remote-release#4870]
Issue Type: <b>Bug</b>

Steps: Ctrl+Shift+P -> Remote-ssh:connect to host

Expected Results: A window pops up so I can choose which host to connect to.

Actual Results: The window didn't pop out. And an alert from vscode on the bottom right conner which says "Command 'Remote-SSH: Connect to Host...' resulted in an error (Running the contributed command: 'opensshremotes.openEmptyWindow' failed.)"

Extension version: 0.50.1
VS Code version: Code 1.55.1 (08a217c4d27a02a5bcde898fd7981bda5b49391b, 2021-04-07T15:06:02.360Z)
OS version: Windows_NT x64 10.0.19042
Remote OS version: Linux x64 4.19.128-microsoft-standard

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-8350U CPU @ 1.70GHz (8 x 1896)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>protected_video_decode: enabled<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|7.92GB (0.64GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|

|Item|Value|
|---|---|
|Remote|WSL: Ubuntu-20.04|
|OS|Linux x64 4.19.128-microsoft-standard|
|CPUs|Intel(R) Core(TM) i5-8350U CPU @ 1.70GHz (8 x 1895)|
|Memory (System)|6.13GB (5.83GB free)|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383:30185418
vspyt653:30270858
vspor879:30202332
vspor708:30202333
vspor363:30204092
vstry914:30276682
pythonvsdeb440:30248342
pythonvsded773:30248341
pythonvspyt875:30259475
pythonvspyt639:30281910
pythontb:30283811
wslfolderdoc:30282074
vspre833cf:30267465
pythonptprofiler:30281270
vshan820cf:30276953
pythondataviewer:30285071
vscus158cf:30286554

```

</details>

**Comment by bamurtaugh** [comment]
@youralmight The latest version of Remote-SSH is 0.65.2 - would you be able to try with the latest version (since it looks you're trying with 0.50.1)?

**Comment by youralmight** [comment]
Sorry. I updated the extension in vs-code to 0.65.3.
Addition: I tried to reinstall vscode and the remote-ssh(and both local cache) to get around the problem. But it didn't work.
Belowings are the log, the new system Info, A/B Experiments, and screenshot of the vscode alert.

Log:
```
[10:41:39.695] Log Level: 2
[10:41:39.698] remote-ssh@0.65.3
[10:41:39.698] win32 x64
[10:41:39.780] Picking SSH host
```

System Info:
```
CPUs | Intel(R) Core(TM) i5-8350U CPU @ 1.70GHz (8 x 1896)
-- | --
GPU Status | 2d_canvas: enabled gpu_compositing: enabled multiple_raster_threads: enabled_on oop_rasterization: enabled opengl: enabled_on protected_video_decode: enabled rasterization: enabled skia_renderer: enabled_on video_decode: enabled vulkan: disabled_off webgl: enabled webgl2: enabled
Load (avg) |  
Memory (System) | 7.92GB (0.94GB free)
Process Argv | --folder-uri vscode-remote://wsl%2Bubuntu-20.04/mnt/c/syncthing/file/workspace/project/duorat_paper_translating --crash-reporter-id b2189776-f7cd-4036-a951-8667a7ec0970
Screen Reader | no
VM | 0%
```
![image](https://user-images.githubusercontent.com/42952005/114807060-56d18300-9dd8-11eb-960c-b4fc243aa7c0.png)
![image](https://user-images.githubusercontent.com/42952005/114806669-b4190480-9dd7-11eb-8728-db53117c851f.png)

**Comment by youralmight** [comment]
I tried the vscode insider version. Not working.

**Comment by youralmight** [comment]
I deleted my ssh configuration file(.ssh/config) and followed the add-on to add a host, and it's working. 


Maybe some hint when the config file is not good would help much.

**Comment by bamurtaugh** [comment]
How did you previously create your config file (i.e. had you already followed the steps from Remote-SSH to add a host, but there was potentially something missing/incorrect about the host you added, or the config file was in an incorrect location)?

In order to get Remote-SSH working, did you need to move your config file to a new place? Did you have a blank config file with no hosts? Did you have hosts in a config file already, and you didn't move it or remove previous hosts, but just added a new one through the prompts from Remote-SSH? 

Want to make sure I understand the exact steps of what was incorrect or missing in the config file and what step(s) fixed it.

**Comment by youralmight** [comment]
A1: Firstly I created the config file following the prompt. But then I edited it in vscode editor.

A2: Yesterday I wonder if the reason why I can't connect is the config file. So I moved the config file to some other path(where remotessh doesn't know). Then I tried to connect with no config file and followed the prompt to create a new config file. Then It works. The old config file has some hosts in it.
